lib/modules/manager/sbt/extract.ts

```diff
@@ -265,6 +301,9 @@ function registryUrlHandler(ctx: Ctx): Ctx {
     if (dep.depType === 'plugin') {
       dep.registryUrls.push(SBT_PLUGINS_REPO);
     }
+    if (dep.depName === 'sbt/sbt') {
+      dep.registryUrls = [GITHUB_TAGS_REPO];
+    }
```

Comment:
```suggestion
```
remove, so it's using the datasource default

Reply:
Will double check but without it I think it was failing as the Maven registry is forced few lines below.